templates/game/index.html.twig

```diff
@@ -1,32 +1,83 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Hello GameController!{% endblock %}
+{% block title %}Hello GameController!
+{% endblock %}
 
 {% block stylesheets %}
-    {{ parent() }}
+{{ parent() }}
 
-    <link href="{{ asset('css/game.css') }}" rel="stylesheet"/>
+<link href="{{ asset('css/game.css') }}" rel="stylesheet" />
 {% endblock %}
 
 {% block body %}
 
 <form action="{{action}}" method="POST" class="game-container">
+
+    {% if turnStatus == 'add_marble' %}
```

Comment:
State machine is better than multiple condition in front end. Using condition when we have short time is good compromise ;)